### DISCUSSION TITLE
Slack help!

### DISCUSSION THREAD

**Discussion by mmowris** [johnste/finicky#229] [General]
**Have you tried creating configuration for this application?**
I've gotten all my "_somecompany.salack.com_" links to open in desktop Slack, but I can't get parameters to pass through.. how can I achieve this?

**What application are you trying to set up**
Any Slack link clicked opens to the conversation/thread/channel etc instead of only opening Slack

**Comment by samtgarson** [comment]
@mmowris did you ever figure this out?

**Reply by montemartin** [reply]
This thread has a few scripts for this: https://github.com/johnste/finicky/issues/96

The last one looks like it should handle most use cases.

**Reply by bric3** [reply]
Some time ago I rewrote the slack integration and was able to cover more cases. This gist has much more, so you may need to pick things that interest you.

https://gist.github.com/bric3/f1863ed36d6bb57d46724e522c544d75

I have largely rewritten this for Finicky v4, but I'll wait for the Finicky 4 release to publish a new version.

By the way this issue has been moved to a discussion see #370
